src/main/java/uk/gov/hmcts/ccd/domain/service/validate/ValidateQuestionableActivity.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.ccd.domain.service.validate;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.collections.MapUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.hmcts.ccd.domain.model.definition.CaseDetails;
+
+import java.util.Map;
+
+public class ValidateQuestionableActivity {
+
+    private static final Logger LOG = LoggerFactory.getLogger(ValidateQuestionableActivity.class);
+    private static final String EMPTY_COLLECTION_VALUE = "\"value\":{}";
+
+    private ValidateQuestionableActivity() {
+    }
+
+    public static Boolean validateEmptyValuedCollections(CaseDetails caseDetails) {
+        if (caseDetails != null) {
+            Map<String, JsonNode> caseData = caseDetails.getData();
+            if (!MapUtils.isEmpty(caseData)) {
+                String data = jsonNodeToString(caseData);
+                if (data.contains(EMPTY_COLLECTION_VALUE)) {
+                    LOG.warn(String.format(
```

Comment:
also print the full stack trace with a debug log, and lower the log level to debug for this particular class.

Reply:
Please also print the full json paths to suspected collection fields to identify where the collection is detected inside the case object. Let's also remember that there may be more multiple fields of this sort in the case date. So, we should report all such json paths to a suspected collection content.

Reply:
Done